changelog/atomicLoad-return-types.dd

```diff
@@ -0,0 +1,45 @@
+`core.atomic.atomicLoad`'s returns types have changed for aggregate types that have indirections.
+
+$(REF atomicLoad, core,atomic) used to strip the `shared` qualifier off too
+eagerly. When an aggregate type has a "head" and a "tail", connected by an
+indirection, then `atomicLoad` used to strip `shared` off the tail. That was a
+bug ($(BUGZILLA 16230)). `atomicLoad` only loads the head. The tail remains in
+shared memory, and must keep the `shared` qualifier.
+
+When loading a struct that contains indirections, `atomicLoad` now returns a
+wrapper that provides getters which return properly typed values.
+
+When loading a class reference, `atomicLoad` now leaves the `shared` qualifier
+on.
+
+---
```

Comment:
Spurious code block delimiter

Reply:
Great catch! Do we have a bugzilla for the missing line number in the error message "Error: unmatched --- in DDoc comment"?